scripts/load_test_data_cypress.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/env bash
+proj_dir=$(readlink -f `dirname $0`/..)
```

Comment:
I guess `-f` can be removed. It just follows symlinks

Reply:
Okay, `-f` is needed, since is outputs the actual path. 

Unfortunately I haven't found a good portable way. There is however a greadlink in homebrew coreutils. Is that an option?

Reply:
I've added a suggestion that works for me in here https://github.com/bakdata/conquery/pull/2400 (simple avoids using readlink)